R/ui_page_heading.R

```diff
@@ -26,7 +26,40 @@ page_heading <- function(pretitle = "Page pretitle", title = "Page title", ...){
         ),
         tags$div(
           class = "col-auto ms-auto d-print-none",
-          ...
+          tags$div(
+            class = "btn-list",
+            tags$a(
+              href = "https://storage.googleapis.com/public-timor-dev/data_report.pdf",
+              target = "_blank",
+              class = "btn btn-primary d-none d-sm-inline-block",
+              tags$svg(
+                xmlns = "http://www.w3.org/2000/svg",
+                class = "icon",
+                width = "24",
+                height = "24",
+                viewbox = "0 0 24 24",
+                `stroke-width` = "2",
+                stroke = "currentColor",
+                fill = "none",
+                `stroke-linecap` = "round",
+                `stroke-linejoin` = "round",
+                tags$path(
+                  stroke = "none",
+                  d = "M0 0h24v24H0z",
+                  fill = "none"
+                ),
+                tags$path(d = "M4 17v2a2 2 0 0 0 2 2h12a2 2 0 0 0 2 -2v-2"),
+                tags$polyline(points = "7 11 12 16 17 11"),
+                tags$line(
+                  x1 = "12",
+                  y1 = "4",
+                  x2 = "12",
+                  y2 = "16"
+                )
```

Comment:
Would be good to put this into an `icon_download()` function so that it can be easily reused. See for example `icon_gear()` for a template.